libsolidity/parsing/Parser.cpp

```diff
@@ -352,21 +352,29 @@ Parser::FunctionHeaderParserResult Parser::parseFunctionHeader(bool _forceEmptyN
 				m_scanner->peekNextToken() == Token::Semicolon ||
 				m_scanner->peekNextToken() == Token::Assign
 			)
-				// Variable declaration, break here.
-				break;
+				break; // Variable declaration, break here.
```

Comment:
I think the preferred style is the previous one. Can you please make sure all your changes follow that?

Reply:
Done.